src/ui/public/accessibility/__tests__/kbn_accessible_click.js

```diff
@@ -0,0 +1,120 @@
+import angular from 'angular';
+import sinon from 'auto-release-sinon';
+import expect from 'expect.js';
+import ngMock from 'ng_mock';
+import '../kbn_accessible_click';
+import {
+  ENTER_KEY,
+  SPACE_KEY,
+} from '../accessible_click_keys';
+
+describe('kbnAccessibleClick directive', () => {
+  let $compile;
+  let $rootScope;
+
+  beforeEach(ngMock.module('kibana'));
+
+  beforeEach(ngMock.inject(function (_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+  }));
+
+  describe('throws an error', () => {
+    it('when the element is a button', () => {
+      const html = `<button kbn-accessible-click></button>`;
+      expect(() => {
+        $compile(html)($rootScope);
+      }).to.throwError(`kbnAccessibleClick doesn't need to be used on a button.`);
```

Comment:
Damn, we have tests elsewhere that pass strings into `throwError`. Thanks for letting me know.